src/vosk/util.ts

```diff
@@ -0,0 +1,55 @@
+import path from 'path';
+import os from 'os';
+import fs from 'fs';
+import { OperatingSystems } from '../sharedTypes';
+import { operatingSystemDllFilePaths } from './helpers';
+
+/**
+ * Returns the path to the vosk DLL library for all OS's
+ */
+export const getBaseDLLPath = () => {
+  // Path is different in dev than in production
+  const prodPath =
```

Comment:
`dllPath`? 'prod' is only true if it's production but this also has a value in development env

Reply:
I'm a bit confused. I understand its a bit hacky but basically just want one path for production and one for development.
This logic checks if the variable exists, which it wouldn't in a proper production app and checks if its in 'development'. I can change it to just check if the variable exists if you like.
Also,
This logic will hopefully be removed in future PR's when we download the model at runtime so it will be the same for prod and development. I am currently working on this atm.

Reply:
just suggested changing the variable name since it's not a `prodPath` if it's not prod

Reply:
Oh I see. Fixed!